_ap/18/index.adoc

```diff
@@ -3,76 +3,194 @@ num: 18
+
+Cross-project integrations are facilitated and https://github.com/microsoft/kiota/discussions/2154[healthy conversations] are initiated on both (RH and Microsoft) sides.
+
+==== Better design/architecture
+
+Kiota design is extremely principled and follows https://microsoft.github.io/kiota/extending/[good and detailed guidelines] throughout the project.
+The project leadership shows broad and deep experience in building successful code generators, which are pragmatic and extensible.
+
+==== Better developer experience
+
+The project's architectural design decisions are supported and embraced throughout all of the different implementations so that the user is going to have a consistent and uniform experience while using the generated code across different languages.
+Additionally, the fluent code style of the generated clients provides a better experience for consumers, resulting in higher levels of developer joy.
+
+==== Maintenance
+
+Experience shows that issues are answered within hours, PRs are continuously accepted and merged, and guidance is provided when needed.
+
+Transitive dependencies are kept up to date in a timely manner.
+
+Improvements and bugs are continuously integrated into both the generator and the auxiliary libraries' repositories.
+
+==== Swappable implementation libraries
+
+The design of Kiota allows consumers of the code generator to provide their own https://microsoft.github.io/kiota/extending/corelibrary.html[“opinionated” implementations] of core generated client functionality.
+This is done via a set of interfaces to core functionality that all generated clients leverage (e.g. a Java HTTP client interface).  Default implementations of these interfaces are provided, but alternatives can easily be created and used.
+
+Specifically for the Red Hat use case, we are free to roll out our own implementations of the core libraries to better support Red Hat products and strategy (e.g. Quarkus specific implementations of certain functionality).
+
+Creating alternatives for the provided default libraries is doable in a very reasonable amount of time.
+https://github.com/andreaTP/kiota-utils/blob/1a7dcecd92ce8794fe479e9c37c71d5733a39ff6/http/jdk-http/src/main/java/io/apicurio/kiota/http/JdkHttpRequestAdapter.java#L1[Here] you can find an alternative implementation of the Java HttpClient interface that is based on the client embedded in JDK 11+ (vs. the out-of-the-box implementation that is based on OkHttp).
+
+In the long term, we expect that most of the bug fixes are going to happen in those shared libraries, and being able to maintain and release them at our own pace is a key element for success.
+
+==== Compatibility matrix
+
+Kiota https://github.com/microsoft/kiota/issues/2020[compatibility matrix] covers the current Red Hat use-cases, specifically:
+
+* Java
+* Go
+* Python
+* TypeScript
```

Comment:
```suggestion
* JavaScript
```

typo?

Reply:
line just below is TypeScript, so TypeScript was listed twice

Reply:
thanks for the suggestion!
This is actually deliberate, even the current "js-sdk" is producing TS:
https://github.com/redhat-developer/app-services-sdk-js